presto-raptor/src/test/java/com/facebook/presto/raptor/storage/organization/TestCompactionSetCreator.java

```diff
@@ -16,13 +16,15 @@
 import com.facebook.presto.raptor.metadata.Table;
 import com.facebook.presto.spi.type.Type;
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
```

Comment:
Can you add a test case to `TestCompactionSetCreator` to test the behavior around shard with delta?